app/src/main/res/layout/fragment_register.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/reg"
+    android:orientation="vertical"
+    >
+
+    <ImageView
+        android:layout_width="@dimen/width_100dp"
+        android:layout_height="@dimen/hieght_100dp"
+        android:layout_gravity="center"
+        android:src="@drawable/pic" />
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:paddingBottom="@dimen/pad_bott_20dp"
+        android:text="@string/register2"
+        android:textColor="@color/color_FFF5F5F5"
+        android:textSize="@dimen/txt_size_25sp"
+
+
+        />
+
+    <EditText
+        android:id="@+id/name"
+        android:layout_width="@dimen/name_width_250dp"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:drawableLeft="@drawable/ic_user_name"
+        android:hint="@string/username"
+        android:textColorHint="@color/color_FFF7F1F4"
+        android:inputType="text"
+        android:padding="@dimen/name_padding"
+        android:textSize="13sp"
+        android:layout_marginLeft="@dimen/name_marginleft"
+        android:backgroundTint="@color/color_FFF8F7F7"
+
+
+        />
+
+    <EditText
```

Comment:
validate against user 
username is not empty 
mail is correct 
password is more than 6 characters and match confirm password